src/main/frontend/src/features/contactus/ContactUs.js

```diff
@@ -26,6 +26,10 @@ const GetInTouchFrame = styled.div`
   background-color: #FFF;
   margin: -350px 10vw 50px 10vw;
   box-shadow: 0 5px 6px -6px black;
+
+  @media (max-width: 768px) {
```

Comment:
There is a util to get the viewport configurations for media queries in _utils/viewport.js_ 

Example:
const Container = styled.div`
  display: flex;
  justify-content: center;
  align-items: center;
  flex-direction: column;
  padding: 20px;
  margin-bottom: 40px;

  @media ${device.laptop} { 
    padding: 100px 50px;
  }
`;

You can find more examples in HomepageIntro.js